CMakeLists.txt

```diff
@@ -7,9 +7,24 @@ project(cb_v2g
 	LANGUAGES C CXX
 )
 
-find_package(everest-cmake 0.1 REQUIRED
+find_package(everest-cmake 0.5
     PATHS ../everest-cmake
+    NO_DEFAULT_PATH
 )
+find_package(everest-cmake 0.5)
```

Comment:
Do we need both `find_package()`, so that both the "neighboring" workspace directory and a "global" installation are checked?

Reply:
Okay for me if tested. 😉